Sources/NIO/Linux.swift

```diff
@@ -71,12 +71,12 @@ internal enum Epoll {
     public static let EPOLL_CTL_DEL: CInt = numericCast(CNIOLinux.EPOLL_CTL_DEL)
 
     #if os(Android)
-    public static let EPOLLIN: CUnsignedInt = numericCast(CNIOLinux.EPOLLIN)
-    public static let EPOLLOUT: CUnsignedInt = numericCast(CNIOLinux.EPOLLOUT)
-    public static let EPOLLERR: CUnsignedInt = numericCast(CNIOLinux.EPOLLERR)
-    public static let EPOLLRDHUP: CUnsignedInt = numericCast(CNIOLinux.EPOLLRDHUP)
-    public static let EPOLLHUP: CUnsignedInt = numericCast(CNIOLinux.EPOLLHUP)
-    public static let EPOLLET: CUnsignedInt = numericCast(CNIOLinux.EPOLLET)
+    public static let EPOLLIN: CUnsignedInt = 1 //numericCast(CNIOLinux.EPOLLIN)
+    public static let EPOLLOUT: CUnsignedInt = 4 //numericCast(CNIOLinux.EPOLLOUT)
+    public static let EPOLLERR: CUnsignedInt = 8 // numericCast(CNIOLinux.EPOLLERR)
+    public static let EPOLLRDHUP: CUnsignedInt = 8192 //numericCast(CNIOLinux.EPOLLRDHUP)
+    public static let EPOLLHUP: CUnsignedInt = 16 //numericCast(CNIOLinux.EPOLLHUP)
+    public static let EPOLLET: CUnsignedInt = 2147483648 //numericCast(CNIOLinux.EPOLLET)
```

Comment:
I'm very nervous about removing these. Can we define the appropriate values in `CNIOLinux` conditionally on Android instead?

Reply:
Oh, these are only defined for Android before my pull, see the `#if` above.